.github/workflows/codeql-analysis.yml

```
@@ -44,7 +44,7 @@
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@1b549b9259bda1cb5ddde3b41741a82a2d15a841 # v3.28.13
+      uses: github/codeql-action/init@45775bd8235c68ba998cffa5171334d58593da47 # v3.28.15
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)